RTCPeerConnection.js

```diff
@@ -110,14 +112,38 @@ export default class RTCPeerConnection extends EventTarget(PEER_CONNECTION_EVENT
       WebRTCModule.peerConnectionRemoveStream(stream._reactTag, this._peerConnectionId);
   }
 
+  addTransceiver(source: 'audio' |'video' | MediaStreamTrack, init) {
+    return new Promise((resolve, reject) => {
```

Comment:
I'm working on an `addTransceiver` implementation in `cordova-plugin-iosrtc` and have been referencing your work here for inspiration.  I noticed that you are returning a Promise here, rather than synchronously returning a transceiver.  Considering the necessary async call to the backend, it makes sense that you had to do it asynchronously, but I'm wondering if anyone has discussed the impact to client libraries.  For example, `mediasoup-client` calls `addTransceiver` and keeps a hold of the returned transceiver (in all of its implementations except ReactNative, ironically).  I'm sure other libraries that use `addTransceiver` take a similar approach.  How would developers bridge the gap between those libraries and this implementation?